src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (number) => {
+    if (number === undefined) {
```

Comment:
Replace `number === undefined` with `!number`

Reply:
But it will cause errors if someone put 0 (zero) to the chain. e.g. adder(4)(0)( ). Doesn't it?